src/rpc/mining.cpp

```diff
@@ -336,12 +337,13 @@ std::string gbt_vb_name(const Consensus::DeploymentPos pos) {
 
 UniValue testproposedblock(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() != 1)
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
             "testproposedblock \"blockhex\"\n"
             "\nChecks a block proposal for validity, and that it extends chaintip\n"
             "\nArguments:\n"
             "1. \"blockhex\"    (string, required) The hex-encoded block from getnewblockhex\n"
+            "2. \"acceptnonstd\" (bool, optional) If set true, returns error if block contains non-standard transaction. Default is set via `-acceptnonstdtxn`\n"
```

Comment:
-acceptnonstdtxn is not the default because right now it can overwrite what the testproposedblock argument. If the argument is true, but -acceptnonstxn=false then the standardness check is executed in testproposedblock.

Reply:
Oh, are you simply saying the description here is backwards? `If set false`

Reply:
Oh, I didn't notice that. But I was talking about the second sentence: the `-acceptnonstdtxn` option overrides  the `acceptnonstd` argument and is therefore more than just a default.